compiler/circle-mpqsolver/src/BisectionSolver.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __MPQSOLVER_BISECTION_SOLVER_H__
+#define __MPQSOLVER_BISECTION_SOLVER_H__
+
+#include <luci/IR/Module.h>
+
+#include <memory>
+#include <string>
+
+namespace mpqsolver
+{
+
+class BisectionSolver final
+{
+
+public:
+  struct Options
+  {
+    enum class Q16AtInput
+    {
+      Q16AtInput_Auto,
+      Q16AtInput_True,
+      Q16AtInput_False
+    };
```

Comment:
Prefixes are duplicates. How about this one?
```suggestion
    enum class Q16AtInput
    {
      Auto,
      True,
      False,
    };
```

Reply:
Thank you. Seems quite reasonable!